sdks/java/extensions/sql/jdbc/src/test/java/org/apache/beam/sdk/extensions/sql/jdbc/BeamSqlLineIT.java

```diff
@@ -198,12 +202,15 @@ private String taxiRideJSON(
     return objectNode.toString();
   }
 
+  /** Suppressing this due to https://github.com/typetools/checker-framework/issues/979. */
+  @SuppressWarnings("return.type.incompatible")
   private Future<List<List<String>>> runQueryInBackground(String[] args) {
     return pool.submit(
         (Callable)
             () -> {
               ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-              BeamSqlLine.runSqlLine(args, null, outputStream, null);
+              InputStream inputStream = new ByteArrayInputStream(new byte[0]);
+              BeamSqlLine.runSqlLine(args, inputStream, outputStream, null);
```

Comment:
I think actually the appropriate fix here is to make `inputStream` `@Nullable` in runSqlLine. It looks like SqlLine handles a null here: https://github.com/julianhyde/sqlline/blob/sqlline-1.4.0/src/main/java/sqlline/SqlLine.java#L717-L722

Could you try that instead?

Reply:
I got your point, but it would require to annotate the `inputStream` [here](https://github.com/julianhyde/sqlline/blob/sqlline-1.4.0/src/main/java/sqlline/SqlLine.java#L615) in `begin` method with `@Nullable`, which is outside the project. What could be possible workaround here ?

Reply:
Could we add an annotation to runSqlLine to suppress that warning? That seems preferable to creating InputStream instances that aren't actually used

Reply:
Done. This `ubuntu-latest` Unit-Test is not failing on my local machine.

Reply:
Thank you! yeah the github actions tests on different targets seem to flake pretty often and I'm not sure how to re-run them. I think its safe to ignore.